pkg/apis/minio.min.io/v2/types.go

```diff
@@ -56,15 +56,40 @@ type TenantScheduler struct {
 	Name string `json:"name"`
 }
 
-// S3Features (`s3`) - Object describing which S3 features to enable/disable in the MinIO Tenant. +
+// TenantDomains (`domains`) - List of domains used to access the tenant from outside the kubernetes clusters.
+// this will only configure MinIO for the domains listed, but external DNS configuration is still needed.
+type TenantDomains struct {
+	// List of Domains used by MinIO. This will enable DNS style access to the object store where the bucket name is
+	// inferred from a subdomain in the domain.
+	Minio []string `json:"minio,omitempty"`
+	// List of Domains used by Console
+	Console []string `json:"console,omitempty"`
```

Comment:
don not who is using Console array here, can we only add it when needed?

Reply:
the next PR will use it, but I'll add it on the next PR